docs/developer/channels.mdx

```diff
@@ -0,0 +1,272 @@
+
+Request:
+```graphql
+query {
+  query Channel($id: ID!) {
+    channel(id: "Q2hhbm5lbDo0MzM=") {
+      id
+      isActive
+      name
+      slug
+      currencyCode
+      hasOrders
+    }
+  }
+}
+```
+
+
+Response:
+```json
+{
+  "data": {
+    "channel": {
+      "id": "Q2hhbm5lbDo0MzM=",
+      "isActive": false,
+      "name": "Mobile",
+      "slug": "mobile",
+      "currencyCode": "PLN",
+      "hasOrders": false
+    }
+  }
+```
+
+
+## Creating new channel
+
+Creating new channel is done using [`channelCreate`](developer/api-reference/mutations/channel-create.mdx) mutation.
+
+Request:
+```graphql
+mutation {
+  channelCreate(
+    input: {
+      "currencyCode":"PLN",
+      "name":"Mobile",
+      "slug":"mobile"
+    }
+  ){
+    channel {
+      id
+      isActive
+      name
+      slug
+      currencyCode
+    }
+    errors: channelErrors {
+      code
+      field
+      message
+    }
+  }
+}
+```
+
+Response:
+```json
+{
+  "data": {
+    "channelCreate": {
+      "channel": {
+        "id": "Q2hhbm5lbDo0MzM=",
+        "isActive": false,
+        "name": "Mobile",
+        "slug": "mobile",
+        "currencyCode": "PLN"
+      },
+      "errors": []
+    }
+  }
+}
+```
+
+### Channel list
+
+Because some of the channels may be considered non-public (for example - channel for business partners), non-staff users can not use [`channels`](developer/api-reference/queries/channels.mdx) query. 
+
+Request:
+```graphql
+query {
+  channels {
+    name
+  }
+}
+```
+
+Response:
+```json
+{
+  "data": {
+    "channels": [
+      {
+        "name": "Mobile",
+      },
+      {
+        "name": "Website"
+      },
+    ]
+  }
+}
+```
+
+### Activate / Deactivate channel
+
+If you want to make the channel unavailable for customers, you can change it's status to deactivated using [`channelDeactivate`](developer/api-reference/mutations/channel-deactivate.mdx): 
+
+```graphql
+mutation{
+   channelDeactivate(id: "Q2hhbm5lbDoxMQ=="){
+    channel {
+      name
+      isActive
+    }
+    channelErrors {
+      message
+    }
+  }
+}
+```
+
+Response:
+```json
+{
+  "data": {
+    "channelDeactivate": {
+      "channel": {
+        "name": "Facebook",
+        "isActive": false
+      },
+      "channelErrors": []
+    }
+  }
+}
+```
+
+And to reverse the previous operation use [`channelActivate`](developer/api-reference/mutations/channel-activate.mdx):
+
+```graphql
+mutation{
+  channelActivate(id: "Q2hhbm5lbDoxMQ=="){
+    channel {
+      name
+      isActive
+    }
+    channelErrors {
+      message
+    }
+  }
+}
+```
+
+Response:
+```json
+{
+  "data": {
+    "channelDeactivate": {
+      "channel": {
+        "name": "Facebook",
+        "isActive": true
+      },
+      "channelErrors": []
+    }
+  }
+}
+```
+
+## Removing a channel
+
+Channels can be removed only when:
+- theres no orders created in them.
+- if there are orders, there's provided `targetChannel` with the same currency as channel you are about to delete.
```

Comment:
Maybe we should add information that all orders related to this channel will be moved to `tergetChannel` ?